crates/tauri-plugin-pilot/js/bridge.snapshot.test.mjs

```
@@ -0,0 +1,125 @@
+      return null;
+    },
+    querySelector() {
+      return null;
+    },
+  };
+  function XMLHttpRequestStub() {}
+  XMLHttpRequestStub.prototype.open = function () {};
+  XMLHttpRequestStub.prototype.send = function () {};
+  globalThis.XMLHttpRequest = XMLHttpRequestStub;
+
+  (0, eval)(BRIDGE_SRC);
```

Biome (2.4.15)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)